sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -164,10 +167,10 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
         auditUserId,
         application.id,
       );
-      let createdRestriction: StudentRestriction | undefined = undefined;
+      let createdRestrictions: StudentRestriction[] | undefined = undefined;
```

Comment:
I believe that `undefined` is not needed here and an empty array will be returned if no items are present.